### PR TITLE
QR Code Implementation

### LINKED ISSUE #52: Xendit QR Codes or QRIS Implementation ?
is there API implementation or helper for Xendit QR Codes or QRIS implementation include the example ?

ref: https://docs.xendit.co/qr-codes/index.html
ref: https://developers.xendit.co/api-reference/#qr-codes

### PR DESCRIPTION
**Pull request by kevindavee**
- Implementation
- Unit Test

Resolves #52

### CODE REVIEW

File: qrcode.go [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+import "time"
+
+// QRCodeTypeEnum constants are the available QR Code type
+type QRCodeTypeEnum string
+
+// This consists the values that QRCodeTypeEnum can have
+const (
+	DynamicQRCode QRCodeTypeEnum = "DYNAMIC"
+	StaticQRCode  QRCodeTypeEnum = "STATIC"
+)
```
Comment (stanleynguyen) [inline qrcode.go:12]: Can we follow Go convention for naming enum?

Comment (stanleynguyen) [reply qrcode.go:12]: Probably just `QRCodeType` would do

Comment (kevindavee) [reply qrcode.go]: Done sir

**Review by slzhffktm — APPROVED** [review]
LGTM

**Pull request merged**